sdk/release_notes.md

```diff
@@ -8,7 +8,8 @@
 
 - Build no longer generates `functions.metadata` if source-generated metadata provider is enabled. (#2974)
 - Fixing `dotnet run` to work on Windows when core tools is installed from NPM (#3127)
+- `Microsoft.Azure.Functions.Worker.Sdk.Generators` bumped to `1.3.6`.
 
-### Microsoft.Azure.Functions.Worker.Sdk.Generators <version>
+### Microsoft.Azure.Functions.Worker.Sdk.Generators 1.3.6
```

Comment:
Do you need to update Microsoft.Azure.Functions.Worker.Sdk version if you are planning for an immediate release?

Reply:
Will do a separate release PR